### ISSUE TITLE
Multiple Errors when running Julius Acoustic Model Creation Code

### ISSUE THREAD

**Issue by resstymanuzon** [julius-speech/julius#85]
Hey guys!
I've spent the last week or so trying to understand JULIUS and implement it into our Speech-To-Text use-case but it's proving to be extremely difficult. I find the Julius book to be incredibly vague and hard to follow. I **desperately** need your help.

When following the acoustic model tutorial provided by Voxforge (http://www.voxforge.org/home/dev/acousticmodels/windows/create/htkjulius/how-to) I keep running into this "ERROR: LoadError: LoadError: syntax: invalid escape sequence" like as displayed in the "[mkclscript.jl](https://raw.githubusercontent.com/VoxForge/develop/master/bin/mkclscript.jl)" file (line 27) mentioned in the "Run Acoustic Model Creation Script" part of the tutorial. The code causing this error is shown below:

`write(hmmlist,"TB 350 \"ST_$phone\_$i\_\" {(\"$phone\",\"*-$phone\+*\",\"$phone\+*\",\"*-$phone\").state[$i]}\n")`

The other codes provided by the tutorial are peppered with so many of these invalid escape sequence errors. I don't even know where to start because I don't even know what the code means. I've already messaged the Voxforge developers and am eagerly waiting for a response. 

Other topics I'm desperately seeking advice for is how to implement the Japanese Model into the Julius software. I _know_ this is a JULIUS forum but perhaps other JULIUS developers may have some insight into this matter. Thank you.

**Comment by Estalhun** [comment]
Hi,

How did you run the script in the CMD?
julia mkclscript.jl ??

I used these and other scipts without any problem.

**Comment by resstymanuzon** [comment]
Hi Estalhun!
Thank you for your comment. 
Please find attached screenshots of the command prompt, the mklscript error and the trainAM error, respectively. Hopefully this makes it clearer. 

Command Prompt:
![error](https://user-images.githubusercontent.com/41427337/46927496-54a3d780-d068-11e8-937e-55727f120d70.PNG)

Mklscript error:
![mklscript error](https://user-images.githubusercontent.com/41427337/46927454-14dcf000-d068-11e8-90f3-12f3f450849d.PNG)

TrainAM error:
![trainam error](https://user-images.githubusercontent.com/41427337/46927456-18707700-d068-11e8-88cc-8a2259fa3528.PNG)

**Comment by Estalhun** [comment]
Check the CRLF at the end of lines.
Check the character encoding.  Do not use UTF-8.
Check the codepage of the CMD.

**Comment by Estalhun** [comment]
And check the non-printable characters too.
